docs/asciidoctorj.adoc

```diff
@@ -39,7 +40,7 @@ ifdef::env-github[]
 image::http://img.shields.io/travis/asciidoctor/asciidoctorj/master.svg[Build Status,90,18,link=https://travis-ci.org/asciidoctor/asciidoctorj]
 endif::[]
 
-ifdef::env-site,env-browser[]
+ifeval::["{toc}" == "macro"]
```

Comment:
I'm saying that I used the wrong attribute in my PR. It should be `toc-placement`, not `toc`.